Update sidekiq readme.md

radamanthus opened the pull request:
Addresses issue #107

mikong commented on cookbooks/sidekiq/readme.md:
```diff
@@ -65,18 +65,20 @@ If you wish to have more than one sidekiq utility instance, name the sidekiq uti
 ## Multi Instance Deploys
 
-By default engineyard will install Redis to your DB instance and if you wish to keep it there rather than the preferred method of creating a utility instance you will need to tell Sidekiq where to find Redis. You can do this by adding a Sidekiq intializer in `config/initializers/sidekiq.rb` with the following information:
+In a clustered environment you need to tell Sidekiq where to find Redis. You can do this by enabling the Redis recipe and adding a Sidekiq intializer in `config/initializers/sidekiq.rb` with the following information:
```
Typo on "Sidekiq intializer".

radamanthus commented:
Fixed the typo and changed `cookbooks/sidekiq/readme.md` to be a simple pointer to the real readme, `examples/sidekiq/cookbooks/custom-sidekiq/README.md`
